features/promotion/receiving_discount/receiving_percentage_discount_promotion_on_order.feature

```diff
@@ -1,7 +1,7 @@
 @receiving_discount
 Feature: Receiving percentage discount promotion on order
     In order to pay proper amount while buying promoted goods
-    As a Customer
+    As a Visitor
```

Comment:
In [this line](https://github.com/Tomanhez/Sylius/blob/fix-features-missing-login-steps/features/promotion/receiving_discount/receiving_percentage_discount_promotion_on_order.feature#L22) there is still a login-related step which should probably be removed